etc/vagrant/etc-bash.bashrc

```diff
@@ -0,0 +1,68 @@
+# System-wide .bashrc file for interactive bash(1) shells.
```

Comment:
what's the point to use two `.bashrc` file (one user-based and one system-wide)?